src/rprocessing/applet/BuiltinApplet.java

```diff
@@ -3,6 +3,19 @@
 import org.renjin.sexp.StringVector;
 
 import processing.core.PApplet;
+import rprocessing.exception.NotFoundException;
```

Comment:
This block of import statements are being added to BuiltinApplet, but not removed from RLangPApplet.